CMakeLists.txt

```diff
@@ -64,14 +64,21 @@ endif()
 include_directories(include ${PROJECT_BINARY_DIR}/include)
 
 configure_file(include/Config.in include/Config.h)
-configure_file(cmake/VecCoreConfig.cmake.in cmake/VecCoreConfig.cmake @ONLY)
+
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+configure_package_config_file(cmake/VecCoreConfig.cmake.in
+  ${CMAKE_CURRENT_BINARY_DIR}/cmake/VecCoreConfig.cmake
+  INSTALL_DESTINATION lib${LIB_SUFFIX}/cmake/${PROJECT_NAME}
+  PATH_VARS CMAKE_INSTALL_INCLUDEDIR CMAKE_INSTALL_DATAROOTDIR
+  )
 configure_file(cmake/VecCoreConfigVersion.cmake.in cmake/VecCoreConfigVersion.cmake @ONLY)
 
-install(FILES "${PROJECT_BINARY_DIR}/include/Config.h" DESTINATION include/${PROJECT_NAME})
-install(DIRECTORY include/${PROJECT_NAME} DESTINATION include)
-install(FILES cmake/FindUMESIMD.cmake DESTINATION lib${LIB_SUFFIX}/cmake/${PROJECT_NAME})
+install(FILES "${PROJECT_BINARY_DIR}/include/Config.h" DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME})
+install(DIRECTORY include/${PROJECT_NAME} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+install(FILES cmake/FindUMESIMD.cmake DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME}/cmake)
```

Comment:
Note that CMake's `find_package` function looks into [specific places](https://cmake.org/cmake/help/latest/command/find_package.html) for the `*Config.cmake` and `Find*.cmake` files, so moving them to `CMAKE_INSTALL_DATAROOTDIR` directly may break things.

Reply:
Actually, the path above seems fine, we just need to adjust for Windows, maybe, but I can do that later.

Reply:
I can add that here if you like, something like (both paths modifiable as needed within the defaults used by `find_package`):

```cmake
set(CMAKE_INSTALL_CMAKEDIR "share/${PROJECT_NAME}/cmake")
if(WIN32)
  set(CMAKE_INSTALL_CMAKEDIR "cmake")
endif()
...
install(FILES cmake/FindUMESIMD.cmake "${CMAKE_INSTALL_CMAKEDIR}")
```

?

Reply:
That's probably ok, but you can also just leave it with the UNIX value only and I will update and test on Windows after I merge.

Reply:
Done - the new use of `VecCore_INSTALL_CMAKEDIR` variable for locations should make the above easy to implement when you need it.